src/components/CheckForm/checkForm.utils.ts

```diff
@@ -57,14 +56,9 @@ function getFirstInput(errs: FieldErrors<CheckFormValues>) {
 }
 
 function searchForSpecialInputs(errKeys: string[] = []) {
-  const probes = errKeys.includes(`probes`) && document.querySelector(`#${PROBES_SELECT_ID} input`);
```

Comment:
I think we could keep this still? Focus on the search input for the probe component?

Reply:
Done 👍